students/geonwoo/users/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name         = models.CharField(max_length=50, unique=True)
+    email        = models.CharField(max_length=50,unique=True)
+    password     = models.CharField(max_length=150,unique=True)
+    phone_number = models.CharField(max_length=50,unique=True)
```

Comment:
unique=True 속성은 정말로 유일해야 되는 값에 붙여주는 속성입니다.
- name은 동명이인이 있을 수 있습니다.
- password도 다른 사람이 같은 비밀번호를 사용할 경우가 있습니다.
- phone_number같은 경우 조금 애매하긴 하지만, unique 속성을 걸어도 괜찮을 것 같습니다.

```suggestion
    name         = models.CharField(max_length=50)
    email        = models.CharField(max_length=50, unique=True)
    password     = models.CharField(max_length=150)
    phone_number = models.CharField(max_length=50, unique=True)
```